lib/src/components/NxAccordion/NxAccordion.scss

```diff
@@ -81,4 +81,14 @@
   border-top: 1px solid #e5e8f4;
   margin: 0 var(--nx-spacing-6x);
   padding: var(--nx-spacing-6x) 0;
+
+  .nx-list:first-child {
+    margin-top: calc(var(--nx-spacing-6x) * -1);
```

Comment:
To keep it connected and make the purpose of this line more clear, the top padding of the accordion content area should probably be extracted into a scss variable

Reply:
> To keep it connected and make the purpose of this line more clear, the top padding of the accordion content area should probably be extracted into a scss variable

Updated it here! 27f715e95bfb758837314828d2958a46ad3e369f